doc/source/examples/dem/granuheap/granuheap.rst

```diff
@@ -0,0 +1,396 @@
+==================================
+Granuheap
+==================================
+
+This example simulates the flow of a cohesive granular material inside a `Granuheap <https://www.granutools.com/en/granuheap>`_ instrument using the discrete element method (DEM). For more detailed information on the concepts and physical meaning of the parameters in Lethe, we strongly recommend visiting `DEM parameters <../../../parameters/dem/dem.html>`_.
+
+
+----------------------------------
+Features
+----------------------------------
+
+- Solvers: ``lethe-particles``
+- Three-dimensional problem
+- Moving solid surface
+- `GMSH <https://gmsh.info/>`_ grids
+- Multi-case simulations
+
+
+----------------------------
+Files Used in This Example
+----------------------------
+
+All files mentioned below are located in the example's folder (``examples/dem/3d-granuheap``).
+
+- Mesh file: ``cylinder.msh`` and ``support.msh``
+- Python file to generate mesh files: ``gmsh.py``
+- Parameters files for the problem: ``granuheap.prm`` and ``granuheap_multicase.prm``
+- Post-processing python files: ``post_processing.py``, ``post_processing_multicase.prm`` and ``post_processing_functions.py``.
+
+-----------------------
+Description of the Case
+-----------------------
+
+This example simulates the flow of wet sand during a Granuheap experiment. First, the packing of particles is generated by inserting particles inside a cylinder. After 1.6s, the cylinder rises, and the particles can flow freely, producing a heap. We compare the numerical results with our experimental results. To remain consistent with the Granuheap experiment, we use the same metrics in our simulations: a cylinder of 0.01m in diameter. For more information aboutt the Granuheap procedures, visit the `Granuheap web page <https://www.granutools.com/en/granuheap>`_. To match our experimental results, we test multiple combinations of parameters in our simulations. We use the work of Hu *et al.* [#hu2022]_ as a basis to choose which contact properties will create those combinations.
+
+-------------------
+Generate Mesh Files
+-------------------
+
+To generate the ``cylinder.msh`` and ``support.msh`` files, we used the ``gmsh.py`` file. The two functions take as input the bottom's position (x0), the radius, the number of points in the circumference, and the height. 
```

Comment:
I would give a better  name than gmsh.py to the file. Like something like: generate_meshes.py or something similar.